### DISCUSSION TITLE
Bug in `GetUnoccupiedPoints`?

### DISCUSSION THREAD

**Discussion by ericphanson** [BattlesnakeOfficial/feedback#212] [Bug Reports]
The `!` in this line doesn't look right: https://github.com/BattlesnakeOfficial/rules/blob/e6e36ce46ffde7d00355ee84bfc9cb74fa3c9368/board.go#L481. I think it should be
```
if i == 0 && includePossibleMoves
```
so that the possible moves are included if `includePossibleMoves==true`.

**Comment by robbles** [comment]
@ericphanson Thanks for pointing this out - I think you might be right!

If you want to make a PR fixing this, I can review it. Otherwise I'll change this sometime in the next week.
